frontend/app_flowy/packages/appflowy_editor/lib/src/render/rich_text/flowy_rich_text.dart

```diff
@@ -164,43 +189,62 @@ class _FlowyRichTextState extends State<FlowyRichText> with Selectable {
     );
   }
 
-  // unused now.
```

Comment:
Will this be used in the upcoming release? I think keeping this part for future use cases would be an option.

Reply:
No. This part of the code will not be executed in the foreseeable future. 😊